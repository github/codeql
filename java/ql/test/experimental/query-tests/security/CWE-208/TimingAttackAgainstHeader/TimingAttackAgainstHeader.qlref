experimental/Security/CWE/CWE-208/TimingAttackAgainstHeader.ql
