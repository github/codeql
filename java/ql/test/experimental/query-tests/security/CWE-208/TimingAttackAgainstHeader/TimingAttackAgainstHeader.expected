edges
nodes
| Test.java:10:27:10:59 | getHeader(...) | semmle.label | getHeader(...) |
subpaths
#select
| Test.java:10:27:10:59 | getHeader(...) | Test.java:10:27:10:59 | getHeader(...) | Test.java:10:27:10:59 | getHeader(...) | Possible timing attack against $@ validation. | Test.java:10:27:10:59 | getHeader(...) | client-supplied token |
